app/controllers/shared/api.php

```diff
@@ -573,8 +578,10 @@
                 $response
                     ->addHeader('Cache-Control', sprintf('private, max-age=%d', $timestamp))
                     ->addHeader('X-Appwrite-Cache', 'hit')
-                    ->setContentType($cacheLog->getAttribute('mimeType'))
-                    ->send($data);
+                    ->setContentType($cacheLog->getAttribute('mimeType'));
+                if (!$isImageTransformation || !$isDisabled) {
```

Comment:
What happens if $isImageTransformation and $isDisabled are both true, where does the response get sent?

Reply:
it goes to the endpoint as cache doesn't send the response.